File: kalendarz2.react/src/common/models/event/event.ts
export default interface event {
  id:number;
  authorId: number;
  title: string;
  description: string;
  location:string;
  participantsEmails:string[]|null;
  startEvent: Date;
  endEvent: Date;
  color:string;
  isRecurring:boolean
}

export const initialState: event = {
  id:-1,
  authorId: -1,
  title: "",
  description: "",
  location:"",
  participantsEmails:[],
  startEvent:  new Date(),
  endEvent: new Date(),
  color: "#0000ff",
  isRecurring:false
};
    

File: Kalendarz2.Infrastructure.EntityFramework/Entities/Event.cs
namespace Kalendarz2.Infrastructure.EntityFramework;

public class Event
{
    public int Id { get; set; }
    public int AuthorId { get; set; }
    public string Title { get; set; }
    public string Location { get; set; }
    public string Description { get; set; }
    public ICollection<Participation> Participants { get; set; }
    public DateTime StartEvent { get; set; }
    public DateTime EndEvent { get; set; }
    public bool IsDeleted { get; set; }
    public bool IsRecurring { get; set; }
    public int Color { get; set; } //color stored as hex number
}